core/embed/trezorhal/stm32f4/common.c

```diff
@@ -122,6 +120,7 @@ __fatal_error(const char *expr, const char *msg, const char *file, int line,
               rev[4]);
 #endif
   term_printf("\nPlease contact Trezor support.\n");
+  display_refresh();
```

Comment:
`term_printf()` calls `display_refresh()` itself -  so it's not needed here, as well as at few other places in this file and `stm32u5/common.c`.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/1e3d5a49abf84dcbde52aab538f3ccc73a2ec16b